src/vendors/autocomplete.js

```diff
@@ -216,14 +225,10 @@ var autoComplete = (function(){
                 }
               }
               that.timer = setTimeout(function(){
-                if(that.sourcePending) {
-                  that.sourcePending.abort();
-                  that.sourcePending = null
-                }
                 that.sourcePending = o.source(val);
                 that.sourcePending.then((source) => {
-                  suggest(source, val);
                   that.sourcePending = null
+                  suggest(source, val);
```

Comment:
`suggest` should NOT be called here if `source == null` (= if the request has been aborted)

Reply:
Seems to work properly.